students/eldonab/lesson03/mailroom.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+#create a list of at least 5 donors.
+donors_list = [["John Smith", 50000, 600000], ["Bill Gates", 100000, 400, 7000], ["Andy Jones", 400000, 600000], ["Hannah Jones", 30000, 40000], ["Anna Allen", 80000,90000, 1000]] 
+# to add a new donation amount to an existing donor's name.
+def add_donation_to_list(res_name):
+    amount = float(input("How much is the new donation amount? "))
+    for donor in donors_list:
+        if res_name in donor:
+            donor.append(amount)
+# to add a new donor name&donation amount:
+def add_new_donor(new_donor):
```

Comment:
some formatting nit picking:
* you should have 2 lines space between module functions
* your comment can be a docstring

```
def add_donation_to_list(res_name):
    pass


def add_new_donor(new_donor):
    """Add a new donor name&donation amount"""
```